superset_patchup/__init__.py

```diff
@@ -1,5 +1,14 @@
 """
 Main init file for superset_patchup
 """
-VERSION = (0, 1, 4)
-__version__ = ".".join(str(v) for v in VERSION)
+from .version import (VERSION, __version__) # noqa
+
+
+# pylint: disable=unused-argument
+def patch_app(superset_app):
```

Comment:
It would be ridiculously cool to name this **ketchup**

Reply:
Ha - yeah that also went through my head.  ¿Que no?

Reply:
I vote for it :)

Reply:
+1 vote from me too